chrome-os-best-practices/lib/main.dart

```diff
@@ -0,0 +1,272 @@
+import 'package:flutter/material.dart';
+
+
+final ThemeData kIOSTheme = new ThemeData(
+  primarySwatch: Colors.orange,
+  primaryColor: Colors.grey[100],
+  primaryColorBrightness: Brightness.light
+);
+final ThemeData kDefaultTheme = new ThemeData(
+  primarySwatch: Colors.purple,
+  accentColor: Colors.orangeAccent
+);
+void main() {
+  runApp(new FriendlychatApp());
+}
+
+class FriendlychatApp extends StatelessWidget {
+  final String name = "Kenneth Ford";
+
+  @override
+  Widget build(BuildContext build) {
+    return new MaterialApp(
+      title: "Friendlychat",
+      theme: Theme.of(build).platform == TargetPlatform.iOS ? kIOSTheme : kDefaultTheme,
+      home: new ChatAppHomePage(title: "Friendlychat"),
+    );
+  }
+}
+
+class ChatAppHomePage extends StatefulWidget {
+  ChatAppHomePage({Key key, this.title}) : super(key: key);
+
+  final String title;
+
+
+  @override
+  _ChatAppHomePageState createState() => _ChatAppHomePageState();
+}
+
+class _ChatAppHomePageState extends State<ChatAppHomePage> {
+  final List<ChatListEntry> chats = new List<ChatListEntry>();
+
+  @override
+  Widget build(BuildContext context) {
+    chats.add(ChatListEntry("Person 1"));
```

Comment:
Don't do this in the build method, as more will be added every time it is run.

If you navigate back and forth from chat window, the list will continue to grow.

Do this in initState instead:

  @override
  void initState() {
    chats.add(ChatListEntry("Person 1"));
    chats.add(ChatListEntry("Person 2"));
    chats.add(ChatListEntry("Person 3"));
    super.initState();
  }